database/index.js

```diff
@@ -1,13 +1,15 @@
 const mongoose = require('mongoose');
+const fs = require('fs');
 const {Schema} = require('mongoose');
-const {Model} = require('mongoose');
+const products = require('./data.js');
 
 const productSchema = new Schema({
-  image0: String,
+  'prime_pic': String,
```

Comment:
Just being picky, I would be consistent with either putting the single quotes on keys or not putting them at all.

Reply:
That's a good point, I originally didn't want quotes at all but you have to use them if you have an underscore... I'll put quotes on all of them.